azure-steps-windows.yml

```diff
@@ -39,11 +40,11 @@ steps:
     If ($(BITS) -eq 32) {
         $env:CFLAGS = "-m32"
         $env:LDFLAGS = "-m32"
-        $env:PATH = "$env:RTOOLS40_HOME\\mingw32\\bin;$env:PATH"
+        $env:PATH = "$env:RTOOLS43_HOME\\mingw32\\bin;$env:PATH"
```

Comment:
If you want to use rtools43 it'd be worth checking that the tools are in the same place.